Source/FlowEditor/Private/Asset/SFlowDiff.cpp

```diff
@@ -6,12 +6,17 @@
 #include "FlowAsset.h"
 
 #include "EdGraphUtilities.h"
+#include "Editor.h"
```

Comment:
Compiling this in UE5.1 had a lot of missing headers for some reason.  

I filled them in.